intercom_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # Changelog
 
+<<<<<<< HEAD
 # 1.3.0
 - Updated `company` parameter inside `updateUser` method.
+=======
+# 1.2.2
+
+- Added method `displayMessages`.
+
+# 1.2.1
+
+- Updated dependencies
+>>>>>>> origin/master
```

Comment:
@mariopepe this doesn't look good, these are the leftovers from the merge, so we need in ideal case to remerge it